Content.Server/Administration/UI/AdminLogsEui.cs

```diff
@@ -0,0 +1,178 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Content.Server.Administration.Logs;
+using Content.Server.Administration.Managers;
+using Content.Server.EUI;
+using Content.Server.GameTicking;
+using Content.Shared.Administration;
+using Content.Shared.Administration.Logs;
+using Content.Shared.Eui;
+using Robust.Shared.GameObjects;
+using Robust.Shared.IoC;
+using static Content.Shared.Administration.AdminLogsEuiMsg;
+
+namespace Content.Server.Administration.UI;
+
+public sealed class AdminLogsEui : BaseEui
+{
+    private const int LogBatchSize = 1000;
```

Comment:
cvar?

Reply:
Done in https://github.com/space-wizards/space-station-14/pull/5419/commits/66ed26f0ff3a2909c07f37fee60dcf87ea006d08